todo-app/src/App.js

```diff
@@ -1,10 +1,49 @@
 import React, { Component } from 'react';
+import Task from './Task';
+import tasks from './task.json';
 
 class App extends Component {
+
+  constructor(props) {
+    super(props);
+
+    // State
+    this.state = {
+      tasks_arr: tasks,
+    };
+
+    this.handleDoneChange = this.handleDoneChange.bind(this);
+  }
+  
+  handleDoneChange(doneValue, taskId) {
+    const updatedtasks = this.state.tasks_arr.map(task => {
+      if (taskId === task.id) {
+        return Object.assign({}, task, {done: doneValue});
+      }
+      return task;
+    });
+    this.setState({
+      tasks_arr: updatedtasks
```

Comment:
Would be nice to see a uniform method for variable/property naming. Underscoring or camel case are both fine.